### ISSUE TITLE
Opensearch Upgrade from 1.2.x to 2.4 Documentation 

### ISSUE THREAD

**Issue by SomanathThilak** [opensearch-project/OpenSearch#5933]
I am doing a upgrade to opensearch version 2.4 . Can someone point me to the documentation for opensearch upgrade/rollback procedure.

I could find document procedure only for upgrade(migration) from elasticsearch to opensearch.

**Comment by dblock** [comment]
I think you're looking for https://opensearch.org/docs/latest/upgrade-to/upgrade-to/. 

https://forum.opensearch.org/ may be a good place to ask more questions! Hope this helps.

**Comment by SomanathThilak** [comment]
The link mentioned is to upgrade from elasticsearch to opensearch . The one i am looking for is from opensearch 1.x to opensearch 2.x

**Comment by dblock** [comment]
@SomanathThilak you're absolutely right, I tried to find it myself and couldn't, going to reopen and move to documentation-website
